public/app/features/explore/LogsVolumePanel.tsx

```diff
@@ -60,14 +60,14 @@ export function LogsVolumePanel(props: Props) {
   if (zoomRatio !== undefined && zoomRatio < 1) {
     zoomLevelInfo = (
       <>
-        <span className={styles.zoomInfo}>Reload logs volume</span>
+        <span className={styles.zoomInfo}>Reload log volume</span>
         <Button size="xs" icon="sync" variant="secondary" onClick={onLoadLogsVolume} />
       </>
     );
```

Comment:
as reported by Accessibility insights, this button is lacking a label. we could use the text next it as a label:

```suggestion
    zoomLevelInfo = (
      <label className={styles.zoomInfo}>
        Reload log volume&nbsp;
        <Button size="xs" icon="sync" variant="secondary" onClick={onLoadLogsVolume} />
      </label>
    );
```

(in this case i'd also add `cursor: pointer` to `styles.zoomInfo`)

Reply:
actually, we should probably be using grafana-ui:
```
zoomLevelInfo = (
      <InlineField className={styles.zoomInfo} label="Reload log volume" transparent>
        <Button size="xs" icon="sync" variant="secondary" onClick={onLoadLogsVolume} id="reload-volume" />
      </InlineField>
    );
 ```

in this case we can also remove `styles.zoomInfo`

Reply:
Good point. Updated in 38f912d246e847c6cf7bc395d0ef20bfca5d6b74